WordPressUtils/src/main/java/org/wordpress/android/util/ImageUtils.java

```diff
@@ -607,9 +608,11 @@ public static String optimizeImage(Context context, String path, int maxImageSiz
      * Note that this method could take time if network url.
      *
      * @param videoPath The path to the video on internet
+     * @param maxWidth the maximumWidth for the thumbnail
+     * @param headers optional: HTTP headers map to add to the request if the video is remote
      * @return the path to the picture on disk
      */
-    public static Bitmap getVideoFrameFromVideo(String videoPath, int maxWidth) {
+    public static Bitmap getVideoFrameFromVideo(String videoPath, int maxWidth, Map<String, String> headers) {
```

Comment:
We might want to override this method instead of modifying the existing version.

```
public static Bitmap getVideoFrameFromVideo(String videoPath, int maxWidth) {
   return getVideoFrameFromVideo(videoPath, maxWidth, new HashMap<String, String>());
}


public static Bitmap getVideoFrameFromVideo(String videoPath, int maxWidth, @NonNull Map<String, String> headers) {
if (headers == null) throw new IllegalArgumentException...

.....
}
```

It'd have two benefits
1. We wouldn't need to update the calling sites which are passing just two parameters
2. We'd be able to remove the `if (headers != null)`, which feels mis-located

Wdyt?

Reply:
sounds good, will update here shortly

Reply:
updated in ca09b14 👍